e2e/validation.go

```diff
@@ -102,5 +116,28 @@ func commonLiveVMValidators() []*LiveVMValidator {
 				"cloud-config.txt",
 			},
 		),
+		// CURL goes to port 443 by default for HTTPS
+		{
+			Description: "check that curl to wireserver fails",
+			Command:     "curl https://168.63.129.16/machine/?comp=goalstate -H 'x-ms-version: 2015-04-05' -s --connect-timeout 4",
+			Asserter: func(code, stdout, stderr string) error {
+				if code != "28" {
+					return fmt.Errorf("validator command terminated with exit code %q but expected code 28 (CURL timeout)", code)
+				}
+				return nil
+			},
+			IsPodNetwork: true,
+		},
+		{
+			Description: "check that curl to wireserver port 32526 fails",
```

Comment:
Added two separate checks, because I couldn't figure out an easy way to run two CURLs and fetch both exit codes using just a one liner. This will likely be modified/refactored once i get around to adding a separate type of pod validator like discussed in other comments here.

Reply:
Maybe should add a third one?

That curl from the same node succeeds.

Reply:
Did you run that CURL while exec'd into the pod from non-host debug DS? If not, then it will work from the node itself, but not from pod network (so those non-host debug pods)

Reply:
It's hard to test "something shouldn't happen".
Environment may change, wireserver can move to another port but our tests still will be happy proving nothing.
I just want to see the wireserver is actually present on the node before testing the access is blocked.

Reply:
I believe we can treat wireserver address/ports as "const", since they are the same all across azure: https://learn.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16.

I agree we could add a check from the node itself, I understand what you meant now, adding third check!